mappings/net/minecraft/util/math/noise/PerlinNoiseSampler.mapping

```diff
@@ -16,7 +16,7 @@ CLASS net/minecraft/class_3756 net/minecraft/util/math/noise/PerlinNoiseSampler
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
-	METHOD method_16449 getGradient (I)I
+	METHOD method_16449 permutate (I)I
```

Comment:
Since this sends an input to the permutation map for an output, can this be called `map`? Also the parameter can be updated to `input` than `hash`

Reply:
Now that I think about it, `permutate` sounds like it takes a whole array. `hash` was wrong or inaccurate as the sequence of the calls in the `sample` method is the hash function.